src/main/java/chess/domain/Board.java

```diff
@@ -0,0 +1,99 @@
+package chess.domain;
+
+import chess.domain.dto.PieceResponse;
+import chess.domain.exception.IllegalPieceMoveException;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Board {
+    public static final Piece EMPTY_PIECE = Piece.empty();
+    private static final int RANK_SIZE = 8;
+    private final Map<Position, Piece> piecePosition = new HashMap<>();
+
+    public Board() {
+        makeBlackPiece();
+        makeWhitePiece();
+        makeEmptyPiece();
+    }
+
+    private void makeBlackPiece() {
+        makePiecesExceptPawns(Color.BLACK, Rank.EIGHT);
+        makePawns(Color.BLACK, Rank.SEVEN);
+    }
+
+    private void makeEmptyPiece() {
```

Comment:
```suggestion
    private T makeBlackPiece() {
        makePiecesExceptPawns(Color.BLACK, Rank.EIGHT);
        makePawns(Color.BLACK, Rank.SEVEN);
    }

    private T makeEmptyPiece() {
```

객체 내부의 private method들을 1차적으로 추상화 된 헬퍼 함수라고 할 수 있는데요...!
메서드의 선언부 중 반환값을 void로 사용하지 않는 것 보다는 무언가를 반환해보면 어떨까요??

Reply:
private 함수들이 반환값을 가지게 만들어봤습니다..!
하지만 어느 함수에서는 void 반환을 가지는 메서드가 생기는 것 같았는데
void함수 즉 등록하는 함수를 최소화 하가 위해서 주신 피드백이 맞을까요??

Reply:
와 너무 좋은 인사이트인 것 같아요!!
생각해보니 제가 했던 것은 그저 메서드 나누기 밖에 뿐이었던 것 같아요 ㅠㅠ
하나의 메서드를 처리하기 위해서 예를 들어 생성자도 될 수 있겠죵??
메서드를 나누는 것이 한 기능을 처리하기 위한 여러 메서드의 협력으로 처리할려할 때
무응답이라면 이것이 객체 지향적일까? 라는 말씀으로 이해했는데 제가 잘 이해했을까요???ㅎㅎ

Reply:
모든 부분을 변경하기 보단 BoardGenerator 부분만 제가 이해한대로 바꿔봤는데 제가 잘 이해한 것이 맞나요??